optaweb-vehicle-routing-frontend/src/routes/Depots.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { WorkInProgress } from 'src/components/WorkInProgress';
+import { IDispatchProps, IStateProps } from 'src/containers/OVR';
+
+export interface IDepotsState {
+  readonly tbc: string;
+}
+type IDepotsProps = IDispatchProps & IStateProps;
+export default class Depots extends React.Component<
+  IDepotsProps,
+  IDepotsState
+> {
+  constructor(props: IDepotsProps) {
+    super(props);
+
+    this.state = {
+      tbc: 'To Be completed',
```

Comment:
Let's avoid unused/temporary/placeholder code. Anything that's merged into master should be completed and working or not merged if it's still in progress. In this case
```js

export default function Depots() {
  return <WorkInProgress />;
}
```
should be enough.